app/static/src/js/all.js

```diff
@@ -0,0 +1,22 @@
+AssessFrontend.initAll = function (options) {
```

Comment:
Nice to see we have a central js file now.

Reply:
just need to be polyfilled

Reply:
I know